patches/net/minecraft/client/renderer/PostPass.java.patch

```diff
@@ -0,0 +1,73 @@
+--- a/net/minecraft/client/renderer/PostPass.java
++++ b/net/minecraft/client/renderer/PostPass.java
+@@ -122,7 +_,54 @@
+     }
+ 
+     @OnlyIn(Dist.CLIENT)
+-    public static record TargetInput(String samplerName, ResourceLocation targetId, boolean depthBuffer, boolean bilinear) implements PostPass.Input {
++    public static record TargetInput(String samplerName, ResourceLocation targetId, BufferType bufferType, boolean bilinear) implements PostPass.Input {
++        public enum BufferType {
```

Comment:
Question: should this be `@OnlyIn` too, or should we move it to another type? This was the suggestion by @embeddedt to combine the separate bools into a single enum

Reply:
Since nobody's responded to this in 3 weeks, I'm going to assume it's fine to leave as-is. If something breaks, we can fix it in a follow-up PR/commit.